src/casp/services/api/routers/cellarium_general_router.py

```diff
@@ -39,7 +39,7 @@ async def get_feature_schemas(_: models.User = Depends(dependencies.authenticate
 
 
 @cellarium_general_router.get("/feature-schema/{schema_name}", response_model=t.List[str])
-async def get_feature_schema_by(schema_name: str):
+async def get_feature_schema_by(schema_name: str, _: models.User = Depends(dependencies.authenticate_user)):
```

Comment:
I imagine this is just a python quirk I don't understand.  What purpose does this serve here?

Reply:
Yep exactly.  It's really more of a FastAPI quirk